Bottom: 701da94896404ff49aa660ac3c4c76e49c9bbe3b
Top:    24d4cbe70a9b7ac06e879df76a8bd04ad0c22499
Author: scott snyder <sss@karma>
Date:   2020-04-05 00:09:30 -0400

Refresh of cad-cah-smbc

---

diff --git a/ComicReader/src/com/blogspot/applications4android/comicreader/comictypes/DailyComic.java b/ComicReader/src/com/blogspot/applications4android/comicreader/comictypes/DailyComic.java
index fd359bb..305d052 100755
--- a/ComicReader/src/com/blogspot/applications4android/comicreader/comictypes/DailyComic.java
+++ b/ComicReader/src/com/blogspot/applications4android/comicreader/comictypes/DailyComic.java
@@ -9,8 +9,6 @@ import com.blogspot.applications4android.comicreader.core.Bound;
 import com.blogspot.applications4android.comicreader.core.Comic;
 import com.blogspot.applications4android.comicreader.core.Strip;
 
-import android.util.Log;
-
 /**
  * This class is useful for all those comics which are released daily and are indexed
  * by calendar date.
